src/modules/position_estimator_inav/CMakeLists.txt

```diff
@@ -31,16 +31,11 @@
 #
 ############################################################################
 
-set(MODULE_CFLAGS )
-if(${OS} STREQUAL "nuttx")
-	list(APPEND MODULE_CFLAGS -Wframe-larger-than=4000)
```

Comment:
I warn automatically when stack is less than frame size. I can revert this
and stack related changes probably got a bit carried away and I'm not sure
how 1-1 stack requirements and frame size are.
On Mar 23, 2016 2:49 PM, "Lorenz Meier" <notifications@github.com> wrote:

> In src/modules/position_estimator_inav/CMakeLists.txt
> <https://github.com/PX4/Firmware/pull/4036#discussion_r57225037>:
>
> > @@ -31,16 +31,11 @@
> >  #
> >  ############################################################################
> >
> > -set(MODULE_CFLAGS )
> > -if(${OS} STREQUAL "nuttx")
> > -	list(APPEND MODULE_CFLAGS -Wframe-larger-than=4000)
>
> Why is not needed any more?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/PX4/Firmware/pull/4036/files/88208d0907471173f9234ebeab3171ca7b187c8b#r57225037>
>